common/src/main/java/com/wynntils/mc/mixin/PlayerMixin.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright © Wynntils 2022.
+ * Copyright © Wynntils 2022-2023.
```

Comment:
please revert this spurious change.

Also, please check your own PRs after you create them, and/or make updates to them! ("self reviewing").

This is a trivial thing to notice and fix yourself, and pushing that burden on the reviewers is borderline arrogance.

Reply:
this was spotless

Reply:
... because i changed something in there applied spotless and then reverted it the spotless change stuck